controllers/keystoneapi_controller.go

```diff
@@ -723,7 +723,10 @@ func (r *KeystoneAPIReconciler) generateServiceConfigMaps(
 
 // reconcileConfigMap -  creates clouds.yaml
 // TODO: most likely should be part of the higher openstack operator
-func (r *KeystoneAPIReconciler) reconcileConfigMap(ctx context.Context, instance *keystonev1.KeystoneAPI) error {
+func (r *KeystoneAPIReconciler) reconcileConfigMap(
```

Comment:
Should this function instead use `lib-common`'s config map functions like the `KeystoneAPI` controller already is here: https://github.com/openstack-k8s-operators/keystone-operator/blob/d3b76057f74302810cf3e299421f9785a3ac3491/controllers/keystoneapi_controller.go#L674-L722?

Reply:
Probably we should. maybe in a follow-up(?). I keep bug fixes and refactoring separate so as not to introduce more bugs.

Reply:
I've updated it to use lib-common instead. The CI failure does not seem related.

Reply:
CI issue resolved.